pkg/products/threescale/reconciler.go

```diff
@@ -1551,3 +1575,23 @@ func (r *Reconciler) deleteConsoleLink(ctx context.Context, serverClient k8sclie
 	}
 	return integreatlyv1alpha1.PhaseCompleted, nil
 }
+
+func (r *Reconciler) reconcileDeploymentConfigs(ctx context.Context, serverClient k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
+
+	ns := global.NamespacePrefix + defaultInstallationNamespace
+
+	for _, name := range threeScaleDeploymentConfigs {
+		deploymentConfig := &appsv1.DeploymentConfig{ObjectMeta: metav1.ObjectMeta{Name: name, Namespace: ns}}
+		err := serverClient.Get(ctx, k8sclient.ObjectKey{Name: name, Namespace: ns}, deploymentConfig)
+
+		if err != nil {
+			logrus.Errorf("failed to get DeploymentConfig %s in namespace %s with error: %s", name, ns, err)
+		}
```

Comment:
```suggestion
```

There's no need for this block of code as the `ReconcileZoneTopologySpreadConstraints` function will retrieve the DC anyways:
https://github.com/integr8ly/integreatly-operator/pull/1219/files#diff-618cf8899a2bdb3f5ff975a8867ba9a5R26-R33

Reply:
Ah great. thanks. I've updated now.